test-framework/junit5/src/main/java/org/apache/camel/quarkus/test/AfterAllCallback.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.test;
+
+import io.quarkus.test.junit.callback.QuarkusTestAfterAllCallback;
+import io.quarkus.test.junit.callback.QuarkusTestContext;
+
+public class AfterAllCallback implements QuarkusTestAfterAllCallback {
+
+    @Override
+    public void afterAll(QuarkusTestContext context) {
+        CamelQuarkusTestSupport testInstance = (CamelQuarkusTestSupport) context.getTestInstance();
```

Comment:
We need to check whether the test instance really is `CamelQuarkusTestSupport`. Otherwise it's impossible to write any `@QuarkusTest` annotated tests without extending `CamelQuarkusTestSupport`.

The same applies to the other callbacks.

Reply:
Fixed. This was the reason of failed `rest-json` test.